### DISCUSSION TITLE
Liver volume

### DISCUSSION THREAD

**Discussion by VeronikaVor2** [Open-Systems-Pharmacology/Forum#1678] [Questions & Problems] [answered]
Hi! I plan to use physiological parameter values from PKSim in my calculations and I have a question about liver compartment volume in a typical human - do I understand correctly that it's 2.38 L? It's a bit bigger than I expected and I followed the link and it's about 1.5 L. May I ask if I correctly understand the attached table?
Thank you in advance!
![liver_pksim](https://github.com/Open-Systems-Pharmacology/Forum/assets/76150517/4f991de5-9290-4620-8fcd-bc73f4c59372)

**Comment by AndreDlm** [comment, marked as answer]
Hi @VeronikaVor2, PK-Sim assumes a density of 1 g7cm³ for all organs, hence kg and L are equivalent. The following two scaling steps are needed to convert the value from the ICRP report to those in PK-Sim: 

The ICRP Report gives a typical liver weight of 1.8 kg for a 30 years-old male individual (Table 2.8, p. 18). However, this weight refers to the port-mortem liver drained of blood. To get the the *in vivo* liver weight including blood, you need to account for the `fraction vascular` which is 0.17 for the liver:

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/da6c479e-44fd-4062-b95e-aad2bb438f90)

So the *in vivo* liver weight including blood is 1.8/(1-0.17) = 2.17 kg

The second step includes an up-scaling of all organs in PK-Sim to obtain the final body weight. The background is that PK-Sim does not consider all organs of the human body (e.g., thyroid, thymus, eyes, and several glands are missing), so the sum of the *in vivo* weights of organs implemented in PK-Sim is approx. 9 % lower than the body weight. To match the total body weight, all organs are thus scaled again by a factor of 1.09 which finally gives you the PK-Sim value: 

1.8/(1-0.17) * 1.09 = **2.36 L**, which is the liver volume for the 50<sup>th</sup> percentile in PK-Sim.

**Reply by VeronikaVor** [reply]
Dear @AndreDlm,
Thank you very much for such a detailed explanation! This helps a lot!
Kind Regards,
Veronika
